src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -107,42 +113,42 @@ public static List<PartyDetails> buildHearingPartiesDetails(SscsCaseData caseDat
                 OtherParty otherParty = ccdOtherParty.getValue();
                 partiesDetails.addAll(buildHearingPartiesPartyDetails(
                         otherParty, otherParty.getRep(), otherParty.getHearingOptions(),
-                        otherParty.getHearingSubtype(), appellant.getId(), null, referenceDataServiceHolder));
+                        otherParty.getHearingSubtype(), null, referenceDataServiceHolder, adjournLanguage));
```

Comment:
Should be null for other parties
```suggestion
                        otherParty.getHearingSubtype(), null, referenceDataServiceHolder, null));
```